apps/gyrokinetic.c

```diff
@@ -560,6 +560,7 @@ gkyl_gyrokinetic_app_apply_ic(gkyl_gyrokinetic_app* app, double t0)
     }
   }
   calc_field_and_apply_bc(app, 0., distf, distf_neut);
+  app->field->is_static = app->field->info.is_static;
```

Comment:
@Maxwell-Rosen did you check that existing regression tests work with these changes? i.e. when is_static is not specified in the input file?

Reply:
When not specified, is_static should be set to false by default. Is there a different / more robust way to set this?